package.json

```diff
@@ -29,6 +29,7 @@
     "html-escape": "^2.0.0",
     "html-minifier": "^3.5.5",
     "js-schema": "^1.0.1",
+    "json-schema-ref-parser": "^4.1.0",
```

Comment:
This can be a devDependency. Also, please remove `js-schema` from the list here.